docs/HOWTO-jp.md

```diff
@@ -0,0 +1,34 @@
+# ハウツー一覧
+
+<div align="right" markdown="1">
+
+*[Read this in other languages](README.md#translations)*
+
+</div>
+
+**Free-Programming-Books`へようこそ!**
+
+私たちは新しい貢献者を歓迎します。GitHub で初めて Pull Request (PR) を作成する人も歓迎します。もしあなたがその一人であるなら、役に立つかもしれないリソースをいくつか紹介しましょう：
+
+* プルリクエストについて](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests)
+* [プルリクエストの作成](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request)
+* [GitHub Hello World](https://docs.github.com/en/get-started/quickstart/hello-world)
+* [YouTube - GitHub Tutorial For Beginners](https://www.youtube.com/watch?v=0fKg7e37bQE)
+* [YouTube - How To Fork A GitHub Repo and Submit A Pull Request](https://www.youtube.com/watch?v=G1I3HF4YWEw)
+* [YouTube - Markdown Crash Course](https://www.youtube.com/watch?v=HUBNt18RFbo)
+
+
+どの貢献者も最初のPRから始めています。だから...私たちの[大きく成長している](https://www.apiseven.com/en/contributor-graph?chart=contributorOverTime&repo=ebookfoundation/free-programming-books)コミュニティに参加しませんか？
+
+<details align="center" markdown="1">
+<summary>ユーザー対時間のグラフを見るにはクリックしてください。</summary>
+
+[![EbookFoundation/free-programming-books's Contributor over time Graph](https://contributor-overtime-api.apiseven.com/contributors-svg?chart=contributorOverTime&repo=ebookfoundation/free-programming-books)](https://www.apiseven.com/en/contributor-graph?chart=contributorOverTime&repo=ebookfoundation/free-programming-books)
+
+[![EbookFoundation/free-programming-books's Monthly Active Contributors graph](https://contributor-overtime-api.apiseven.com/contributors-svg?chart=contributorMonthlyActivity&repo=ebookfoundation/free-programming-books)](https://www.apiseven.com/en/contributor-graph?chart=contributorMonthlyActivity&repo=ebookfoundation/free-programming-books)
+
+</details>
+
+経験豊富なオープンソース貢献者であっても、つまずくことがあるかもしれません。PRを投稿すると、***GitHub Actions*が**リンター**を実行し、しばしば間隔やアルファベット表記に関する小さな問題を見つけます**。緑色のボタンが表示されれば、レビューの準備は完了です。もし表示されない場合は、失敗したチェックの下にある「詳細」をクリックして、リンターが何を気に入らなかったのかを調べ、PR をオープンしたブランチに新しいコミットを追加して問題を修正しましょう。
+
+最後に、もしあなたが追加したいリソースが `Free-Programming-Books` にふさわしいかどうか確信が持てない場合は、[CONTRIBUTING](CONTRIBUTING.md) *([translations](README.md#translations)もあります)* にあるガイドラインに目を通してください。
```

Comment:
I suggest to change "CONTRIBUTING.md" to "CONTRIBUTING-ja.md".

Reply:
Renamed CONTRIBUTING.md to CONTRIBUTING-ja.md